app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str):
+    file = command.split(" ")
+    if file[1] != file[2]:
```

Comment:
you should add a condition for the "cp" input command

Reply:
Sorry, but I don't get it. I have the condition.HELP)